opencti-platform/opencti-graphql/config/default.json

```diff
@@ -188,7 +214,8 @@
     "openerm_url": "",
     "openrm_token": "",
     "openmtd_url": "",
-    "openmtd_token": ""
+    "openmtd_token": "",
+    "xtmhub_url": "https://xtmhub.filigran.io"
```

Comment:
Do we want it easily removable ? I can't recall what we discussed

Reply:
This value is more to be able to align apps between different environments.
i.e. `locahost:3000` > `localhost:3001` or `my-octi-instance.prod.fr` > `xtmhub.filigran.io`